docs-src/cloud/emitted-metrics.md

```diff
@@ -0,0 +1,41 @@
+---
+id: emitted-metrics
+title: What metrics are emitted from Temporal Cloud?
+description: Temporal Cloud emits several metrics.
+sidebar_label: Available metrics
+tags:
+  - explanation
+  - observability
+  - temporal-cloud
+  - reference
+---
+
+Temporal tracks the following metrics for your various Namespaces.
+
+- temporal_cloud_v0_frontend_service_error_count - gRPC errors returned aggregated by operation
```

Comment:
Now that I'm seeing it [live](https://temporal-documentation-git-edu-471-q3-cloud-docs.preview.thundergun.io/cloud/metrics#available-metrics) I'm wondering if we should bold these names 🤔

Reply:
I can take a quick pass at that -